src/components/jobCardList.js

```diff
@@ -14,6 +15,7 @@ export default function JobCardList({ jobs, filters, selectedJob, onSelectJob, o
             isSelected={selectedJob===job}
             onSelectJob={onSelectJob}
             onUnselectJob={onUnselectJob}
+            img={(job.thumbnail)? job.thumbnail: boat}
```

Comment:
Isn't this logic already handled in `jobCard`? Currently if we pass an empty job thumbnail, then jobCard should handle it.